tests/remote_attestation/maa/run_scenario.sh

```diff
@@ -0,0 +1,117 @@
+# sgxlkl_enclave starts and connects to oe_enclave
+# sgxlkl_enclave gets the report from oe_enclave and then gets authentication token from AAD
+# sgxlkl_enclave calls MAA for JWT token and gets, parses and verifies it 
+
+# Run mode is sw or hw
+export SGXLKL_RUN_MODE=$1
+
+# Verify that dependent env variables are available
+if [[ -z $SGXLKL_PREFIX || -z $SGXLKL_ROOT || -z $SGXLKL_RUN_MODE || -z $MAA_CLIENT_ID || -z $MAA_CLIENT_SECRET  || -z $MAA_APP_ID || -z $MAA_ADDR || -z $MAA_ADDR_APP ]]; then
+    echo "Dependent environment variables for this scenario not available."
+    echo "SGXLKL_ROOT: $SGXLKL_ROOT"
+    echo "SGXLKL_PREFIX: $SGXLKL_PREFIX"
+    echo "SGXLKL_RUN_MODE: $SGXLKL_RUN_MODE"
+    exit 1
+fi
+
+export SGXLKL_HOST_IMPORT_ENV="MAA_CLIENT_ID,MAA_CLIENT_SECRET,MAA_APP_ID,MAA_ADDR,MAA_ADDR_APP"
+
+# If openenclave is not installed, install it
+if [[ ! -d "/opt/openenclave" ]]; then
+    home=$(pwd)
+    cd ~ || exit 1
+    sudo rm -rf openenclave/
+    git clone --recursive -b feature/sgx-lkl-support https://github.com/openenclave/openenclave.git
+    cd openenclave || exit 1
+
+    sed -i '/add_subdirectory(tests)/d' CMakeLists.txt
+
+    sudo bash scripts/ansible/install-ansible.sh
+    sudo ansible-playbook scripts/ansible/oe-contributors-acc-setup-no-driver.yml
+
+    mkdir -p build
+    cd build || exit 1
+    cmake -G "Ninja"  -DWITH_EEID=1 ..
+    sudo ninja
+    sudo ninja install
+    exit_code=$?
+    if [[ $exit_code -eq 0 ]]; then
+        echo "OE SDK installed successfully"
+    else
+        echo "OE SDK install failed"
+        exit 1
+    fi
+    cd "$home" || exit 1
+fi
+
+# shellcheck disable=SC1091
+. /opt/openenclave/share/openenclave/openenclaverc
+
+# Set log file that will be shared between two enclaves in the test scenario for stdout and stderr logs
+# In sgx-lkl test framework can have one log file for stdout and stderr. This log file is called with cat to be included in test's output
+# This file is also searched for some key sentences to verify TLS connection and MAA attestation results
+export MAA_TEST1_LOG_FILE="$SGXLKL_ROOT/remote_attestation_maa_test.log"
```

Comment:
We probably shouldn't put the log under the root. Suggegst `$SGXLKL_ROOT/report`